content/microcontrollers/bbc-microbit.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "BBC:Microbit"
+weight: 3
+---
+
+The [BBC micro:bit](https://microbit.org) is a tiny programmable computer designed for learning. It is based on the Nordic Semiconductor [nRF51822](https://www.nordicsemi.com/eng/Products/Bluetooth-low-energy/nRF51822) ARM Cortex MO chip.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | YES |
+| I2C      | YES | YES |
+| ADC      | YES | Not yet |
+| PWM      | YES | Not yet |
```

Comment:
The hardware does not natively support PWM. It can of course be emulated in software.

Reply:
Updated.